### ISSUE TITLE
Convert src/MacOSX/ToDo.html to issues and delete it

### ISSUE THREAD

**Issue by ianfixes** [emaculation/macemu#74]
From the [original project](https://github.com/cebix/macemu)'s [`TODO`](https://github.com/cebix/macemu/blob/0a7fe2eaf8e88a458836c3ec1f382906b253d85a/BasiliskII/TODO):
> Mac OS X:
> - Lots of other stuff. See src/MacOSX/ToDo.html

Create github issues from these list items

**Comment by retokromer** [comment]
I also suggest to use the current `macOS` for the system on which the emulation runs.

**Comment by ianfixes** [comment]
> I also suggest to use the current macOS for the system on which the emulation runs.

If you mean that it doesn't currently run on High Sierra, you should open a separate issue for that.

**Comment by retokromer** [comment]
No, sorry for being unclear (English is not my language). I mean the modern Apple system on which the emulator runs should be called `macOS` in the documentation and not `Mac OS X` or other old naming.

On my end, flavours of Basilisk II run on:

- **Linux:** Ubuntu 18.04.1 LTS and 16.04.5 LTS, and Slackware 14.2
- **Macintosh:** macOS 10.14.3 and 10.13.6
- **Windows 10**
